src/modules/html_templates.js

```diff
@@ -31,13 +46,47 @@ const popUpTemplate = (movie) => {
       </div>
     </div>
     <div class="popup-comments">Comments</div>
+    
+  <form class="form-submit">
+  <div class="form-container">
+  <ul class="comment-list">${commentItems()}</ul>
+   <div class="name-field">
+    <input type="text" id="username" placeholder="Please enter your name">
+    <textarea name="textarea" id="comment" cols="10" rows="5" placeholder="please add a comment"></textarea>
+    <button type="submit">Comment</button>
+    </div>
+    </div>
+  </form>
   </div>
   `;
+
   popUpContainer.style.display = 'block';
   const closePopup = document.querySelector('.close-icon');
   closePopup.addEventListener('click', () => {
     popUpContainer.style.display = 'none';
   });
+
+  const form = document.querySelector('.form-submit');
+
+  form.addEventListener('submit', async (e) => {
+    e.preventDefault();
+
```

Comment:
Please let's remove blank lines from our codebase (including other parts of the code they are existing) unless they are separating large or logical code blocks. This helps to make our codebase look cleaner.